targets/targets.json

```diff
@@ -8778,6 +8777,65 @@
         "mbed_ram_start"        : "0x20000000",
         "mbed_ram_size"         : "0x8000"
     },
+    "NU_PFM_M2351_NS": {
+        "inherits": ["NSPE_Target", "NU_PFM_M2351_CM"],
+        "core": "Cortex-M23-NS",
+        "tfm.level": 1,
```

Comment:
@ccli8  TFM_LVL is already defined in https://github.com/ARMmbed/mbed-os/blob/master/components/TARGET_PSA/TARGET_TFM/mbed_lib.json#L7. What is the need for this?

Reply:
Not sure if the default `TFM_LVL` will change. Set it explicitly for safe.